pilot/lib/pilot/manager.rb

```diff
@@ -20,8 +20,8 @@ def login
       config[:username] ||= CredentialsManager::AppfileConfig.try_fetch_value(:apple_id)
 
       UI.message("Login to App Store Connect (#{config[:username]})")
-      Spaceship::Tunes.login(config[:username])
-      Spaceship::Tunes.select_team
+      @session = Spaceship::Tunes.login(config[:username])
+      @selected_team_id = Spaceship::Tunes.select_team
```

Comment:
Instead of storing off `@session` and `@selected_team_id` here and using these in `build_manager.rb`, you could call `Spaceship::Tunes.client` and `Spaceship::Tunes.client.team_id`